src/events/onReactionRemoveUpdateStarboard.evt.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2025 Inimi | DrHooBs | Incoverse
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import { DrBotEvent, DrBotEventTypeSettings, DrBotEventTypes } from "@src/lib/base/DrBotEvent.js";
+import { Client, Embed, EmbedBuilder, Events, Guild, GuildMemberRoleManager, TextChannel } from "discord.js";
+import { DrBotGlobal } from "@src/interfaces/global.js";
+
+export default class ORRUS extends DrBotEvent {
+    declare global: DrBotGlobal;
+    
+    protected _type: DrBotEventTypes = "discordEvent";
+
+    protected _typeSettings: DrBotEventTypeSettings = {
+        listenerKey: Events.MessageReactionRemove,
+    }
+
+    protected _priority: number = 0;
+    
+
+    public async runEvent(reaction, user, client: Client) {
+        if (global.app.config.starboardEmojiID === "" || global.app.config.starboardEmojiSTR === "" || global.app.config.starboardNumber === 0) {
+            return;
+        }
+        if (reaction.partial) {
+            try {
+                await reaction.fetch();
+                await user.fetch();
+            } catch (error) {
+                console.error("Something went wrong when fetching the message: ", error);
+                return;
+            }
+        }
+        if (reaction.message.channel.name.match(/^(staff|mod|admin)([-_][a-z0-9]+)*$/i)) {
+            return;
+        }
+
+        let embed = new EmbedBuilder()
+            .setAuthor({
+                name: reaction.message.author.username,
+                iconURL: reaction.message.author.displayAvatarURL()
+            })
+            .setDescription(reaction.message.content)
+            .addFields({
+                name: "Source",
+                value: `[Jump to message](${reaction.message.url})`
+            })
+            .setFooter({
+                text: `${reaction.message.id} • ${client.user.username}'s Starboard`
```

Comment:
```suggestion
                text: `${reaction.message.id} • ${client.user.displayName}'${client.user.displayName.toLowerCase().endsWith("s") ? "" : "s"} Starboard`
```

While ``.username`` works, ``.displayName`` is better practice in this case. The other change makes it so that if the name ends with "s", it won't add the "s" after the apostrophe (personal preference and it is what's used across other commands / events. it's better to say consistant)